services/src/main/java/io/grpc/services/HealthStatusManager.java

```diff
@@ -26,6 +26,11 @@
  * created in the constructor of {@code HealthStatusManager}, and it can be retrieved by the
  * {@link #getHealthService()} method.
  * The health status manager can update the health statuses of the server.
+ *
+ * <p>
```

Comment:
What's with this formatting? Don't intent because of the `<p>`. Don't include the `</p>`. End in a period.

```
* <p>The default, empty-string, service name, {@link #SERVICE_NAME_ALL_SERVICES}, is initialized to
* {@link ServingStatus#SERVING}.
```

Reply:
It's what IntelliJ likes.  Seems arbitrary.